report/src/test/java/io/testerra/report/test/ReportUnderTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Testerra
+ *
+ * (C) 2021, Mike Reiche,  T-Systems MMS GmbH, Deutsche Telekom AG
+ *
+ * Deutsche Telekom AG and all other contributors /
+ * copyright owners license this file to you under the Apache
+ * License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.testerra.report.test;
+
+import eu.tsystems.mms.tic.testframework.utils.TimerUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ReportUnderTest extends AbstractReportTest {
+
+    @Test
+    public void test_visitReport() {
+        Assert.assertTrue(true);
```

Comment:
Test does nothing, seems incomplete. Looks like a speed-up loop to me? Should there not be some sort of assertion _after_ the wait?

Reply:
Please ignore this. This is just a placeholder for upcomming ReportNG tests.

Reply:
I mean, placeholders _could_ be labeled, e.g. `// TODO` or `// FIXME`... but very well.